src/main/java/com/jcabi/xml/XSLDocument.java

```diff
@@ -70,7 +70,8 @@
 public final class XSLDocument implements XSL {
 
     /**
-     * Strip spaces.
+     * Strips spaces of whitespace-only text nodes. This will NOT remove
+     * line-breaks between Element nodes.
```

Comment:
this looks rather short. in a few months someone will ask the same question - why not remove line-breaks? and will submit a bug. I think we should put a longer explanation, like you did in the ticket. To avoid confusion.